data/index.js

```diff
@@ -92,6 +94,20 @@ export function registerSelectors( reducerKey, newSelectors ) {
 	selectors[ reducerKey ] = mapValues( newSelectors, createStateSelector );
 }
 
+/**
+ * Registers resolvers.
+ *
+ * @param {string} reducerKey   Part of the state shape to register the
+ *                              resolvers for.
+ * @param {Object} newResolvers Resolvers to register.
+ */
+export function registerResolvers( reducerKey, newResolvers ) {
+	resolvers[ reducerKey ] = mapValues(
+		newResolvers,
+		( resolver ) => ( { ...resolver, fulfill: memize( resolver.fulfill ) } )
```

Comment:
What are we spreading?

Reply:
isFulfilled :) or Right I removed it :P. I guess we could leave to accommodate future options but I'll just remove, it's easy enough to add later.